vuecomponents/bundle/src/main/bundle/tag-selector_es.properties

```diff
@@ -0,0 +1,8 @@
+search_or_add=Buscar o añadir etiqueta
```

Comment:
Should we replace ñ by \u00f1 or it is OK?

Reply:
It seems to work fine this way so I would leave it... It's easier to read and maintain

Reply:
Great then! :)